### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by harshitvyas98** [npruehs/ue4-custom-replay-demo#1]
hey, i downloaded your project and when I tried to open it I got this error here:

Running D:/All software install/Unreal/UE_4.23/Engine/Binaries/DotNET/UnrealBuildTool.exe  -projectfiles -project="D:/ReplayTest/ue4-custom-replay-demo-develop/ue4-custom-replay-demo-develop/CustomReplayDemo/CustomReplayDemo.uproject" -game -rocket -progress -log="D:\ReplayTest\ue4-custom-replay-demo-develop\ue4-custom-replay-demo-develop\CustomReplayDemo/Saved/Logs/UnrealVersionSelector-2021.03.17-09.00.00.log"
Discovering modules, targets and source code for project...
While compiling D:\ReplayTest\ue4-custom-replay-demo-develop\ue4-custom-replay-demo-develop\CustomReplayDemo\Intermediate\Build\BuildRules\CustomReplayDemoModuleRules.dll:
d:\ReplayTest\ue4-custom-replay-demo-develop\ue4-custom-replay-demo-develop\CustomReplayDemo\Source\CustomReplayDemo.Target.cs(11,3) : error CS0103: The name 'DefaultBuildSettings' does not exist in the current context
d:\ReplayTest\ue4-custom-replay-demo-develop\ue4-custom-replay-demo-develop\CustomReplayDemo\Source\CustomReplayDemo.Target.cs(11,26) : error CS0103: The name 'BuildSettingsVersion' does not exist in the current context
d:\ReplayTest\ue4-custom-replay-demo-develop\ue4-custom-replay-demo-develop\CustomReplayDemo\Source\CustomReplayDemoEditor.Target.cs(11,3) : error CS0103: The name 'DefaultBuildSettings' does not exist in the current context
d:\ReplayTest\ue4-custom-replay-demo-develop\ue4-custom-replay-demo-develop\CustomReplayDemo\Source\CustomReplayDemoEditor.Target.cs(11,26) : error CS0103: The name 'BuildSettingsVersion' does not exist in the current context
ERROR: Unable to compile source files.

**Comment by harshitvyas98** [comment]
which version are you using ?

**Comment by npruehs** [comment]
Hey @harshitvyas98 ! As the .uproject file should state, the project has been created with Unreal Engine 4.26.1.

Please note that this is by no means a finished, documented or supported project (yet).

**Comment by harshitvyas98** [comment]
thanks, 
i am trying to decompress the replay file from the unreal engine, can you help me with that?

**Comment by npruehs** [comment]
Unreal basically writes the network stream to a file, as if you'd "record" whatever your client received during a multiplayer game, and (literally) _replays_ that network stream.

**Comment by harshitvyas98** [comment]
yes,
hey, in your project can i get the replay file in json format ?

**Comment by npruehs** [comment]
No. The project focuses on parsing JSON files in Unreal Engine, and applying their state to actors in the game world. It has been created as a prototype for another project, which gets their JSON replays from another (undefined) source.

**Comment by harshitvyas98** [comment]
ok, can you make a project where you can parse replay(unreal engine Demo rec) file to json.
ofcourse I don't have problem in paying you for this.
you can email me at : Harshitvyas98@gmail.com
